includes/Core/Modules/Module.php

```diff
@@ -183,7 +183,6 @@ public function prepare_info_for_js() {
 			'cta'          => $this->cta,
 			'sort'         => $this->order,
 			'homepage'     => $this->homepage,
-			'learnMore'    => $this->learn_more,
```

Comment:
@asvinb let's not remove it here because it seems to be used by other components.